src/axl_io.c

```diff
@@ -17,6 +17,12 @@
 #include <errno.h>
 #include <string.h>
 
+/* fiemap */
+#include <linux/fiemap.h>
+#include <sys/ioctl.h>
+#include <linux/fs.h>
+#include <libgen.h>
+
```

Comment:
Do these includes change which OSes we can support? If they only needed for the BBAPI check, maybe we can wrap them in a config-time option.

Reply:
They're standard headers that should be on any linux system.  I'll wrap it in `#if HAVE_BBAPI...` though, since BBAPI is the only thing that uses it.